charts/stateful-app/Chart.yaml

```diff
@@ -2,14 +2,14 @@ apiVersion: v2
 name: stateful-app
 description: Default StatefulSet Helm Chart
 type: application
-version: 0.16.4
```

Comment:
This really should have been 1.0.0 already actually I think because this is no longer beta.